scripts/ai/strategies/AIDriveStrategyFindBales.lua

```diff
@@ -607,6 +587,46 @@ function AIDriveStrategyFindBales:approachBale()
     end
 end
 
+--- Check if we have the field polygon (as we need it to find the bales) and if we are on or close to the field.
+--- The field polygon generation is started by the job and if we direct start, may not have finished yet.
+--- Once we have it, check the vehicle and start positions.
+function AIDriveStrategyFindBales:isPositionOk()
```

Comment:
Not sure but I would guess this kind of a function is also needed for the combine unload job?

Reply:
Yes, that comes next, just didn't have a chance to address it.

Reply:
@schwiti6190 but really the question is, why do we do this in the strategies an also in the jobs. In the jobs, they have a consequence, they make the job invalid. Then the same check in the strategy (bale finder and combine unloader) just logs, but other than that, lets everything continue. 

So either we don't need that check, or it has to do something in the strategy.